modules/jaeger-config-storage.adoc

```diff
@@ -69,8 +69,8 @@ When the `storage:type` is set to `elasticsearch` but there is no value set for
 
 .Restrictions
 
+* There can be only one Jaeger with self-provisioned Elasticsearch instance per namespace. The Elasticsearch cluster is meant to be dedicated for a single Jaeger instance.
```

Comment:
> ! there is, there are verb
Avoid at the beginning of an independent clause. In most cases rewrite to remove the phrase. For example, instead of "There are three recovery control data sets in the product" write "The product includes three recovery control data sets".